tools/dump_tool/ramdumpParser.py

```diff
@@ -563,6 +563,10 @@ def main():
 	print('*************************************************************')
 	print('')
 
+	# Read config information
+	fd = open(config_path, 'r')
+	data = fd.read()
+	fd.close()
```

Comment:
Let us move these lines down into the try statement. 
Because, we dont need to open the config file at this point. If any of the following conditions return false, then it would be waste to read the config here.